bigbluebutton-html5/imports/api/guest-users/server/eventHandlers.js

```diff
@@ -1,6 +1,8 @@
 import RedisPubSub from '/imports/startup/server/redis';
 import handleGuestApproved from './handlers/guestApproved';
+import { processForHTML5ServerOnly } from '/imports/api/common/server/helpers';
 import handleGuestsWaitingForApproval from './handlers/guestsWaitingForApproval';
 
+RedisPubSub.on('GetGuestsWaitingApprovalRespMsg', processForHTML5ServerOnly(handleGuestsWaitingForApproval));
```

Comment:
Would be better to export the `handleGuestsWaitingForApproval` with the enhancer `processForHTML5ServerOnly` already applied (if that's make sense)

Reply:
doesn't make sens in this case the same function is used to `GetGuestsWaitingApprovalRespMsg` and `GuestsWaitingForApprovalEvtMsg` and  would triggered unnecessary in second message